### PR TITLE
Added the 'core.tcp_info' command to the exporter

### PR DESCRIPTION
**Pull request by karstenjakobsen**
The `kamcmd core.tcp_info` command will also retrieve some TLS connection stats. This is useful if you need information regarding TCP and TLS connections. Example:

```
{
	readers: 8
	max_connections: 4096
	max_tls_connections: 4096
	opened_connections: 595
	opened_tls_connections: 595
	write_queued_bytes: 0
}
```

### CODE REVIEW

**Comment by florentchauveau** [comment]
Hello @karstenjakobsen!

Thank you for this. I've merged the "tls.info" before, could you rebase and resubmit?

Best

**Comment by florentchauveau** [comment]
Also, if `core.tcp_info` is available without loading a specific module, it should be safe to add it as a default collector in `main.go`.

What do you think?

Don't forget to update the README.md, the section about default metrics, if you think we can enable it by default :)

File: README.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ Flags:
                              URI on which to scrape kamailio. E.g.
                              "unix:/var/run/kamailio/kamailio_ctl" or
                              "tcp://localhost:2049"
-  -m, --kamailio.methods="tm.stats,sl.stats,core.shmmem,core.uptime"
+  -m, --kamailio.methods="tm.stats,sl.stats,core.shmmem,core.uptime,core.tcp_info"
```
Comment (florentchauveau) [inline README.md:37]: This is the default metrics, so you should update `main.go` as well

**Comment by karstenjakobsen** [comment]
I have fixed conflicts and added `core.tcp_info` to `main.go`

File: collector.go [hunk 2]
```diff
@@ -424,7 +442,11 @@ func (c *Collector) scrapeMethod(method string) (map[string][]MetricValue, error
 		fallthrough
 	case "core.shmmem":
 		fallthrough
+<<<<<<< HEAD
+	case "core.tcp_info":
+=======
 	case "dlg.stats_active":
+>>>>>>> master
```
Comment (florentchauveau) [inline collector.go]: @karstenjakobsen you have an unresolved conflict here

Comment (karstenjakobsen) [reply collector.go]: Omg - i missed that one - fixed

**Pull request merged**